### PR TITLE
Chore: Update PR template to ensure design review performed and restructure checklist

### PR DESCRIPTION
**Pull request by mattvot**
## What

Created a single checklist of pre-submission items and added one to ensure a design review is performed if necessary.

Removed unnecessary headers and used comments to hide item guidance.

## How to review

Read the PR template and check the changes make sense.

## Related PRs

[Same work in the API repo](https://github.com/govuk-one-login/authentication-api/pull/4245)

### CODE REVIEW

**Comment by gtvj** [comment]
Hi @mattvot - just wondering if the two UCD review items might be combined ("changes to the user interface have been demonstrated" and "a design review has been performed") as they seem broadly similar. Apologies if I'm missing some context here.

**Comment by gtvj** [comment]
Hi @mattvot - I'm wondering if maybe we should change the references to "...colleague in your team" to "...colleague in the Authentication Team" to clarify that the review needs to be conducted by UCD and PA collagues in our team (just conscious that PRs are sometimes raised, approved and then merged by other teams).

**Comment by mattvot** [comment]
Hi @gtvj 

> just wondering if the two UCD review items might be combined ("changes to the user interface have been demonstrated" and "a design review has been performed") as they seem broadly similar. Apologies if I'm missing some context here.

I think the intention with them is slightly different. The first is about making sure that when you are making changes that are different from the design (be it hidden by feature flagged or not), that UCD know about it and can update the design files. The seconds is more about a final check before something is seen in live in production that it passed a design review.

> I'm wondering if maybe we should change the references to "...colleague in your team" to "...colleague in the Authentication Team" to clarify that the review needs to be conducted by UCD and PA collagues in our team (just conscious that PRs in this repo are sometimes raised, approved and then merged by other teams).

Sure, can do 👍

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -29,39 +27,41 @@ Include screenshots where possible, including those representing error states. H
...
 
-## Acceptance tests have been updated
+Delete this item if the PR does not change any UI or user journeys.
+-->
+- [ ] Performance analyst has been informed of the change.
 
-- [ ] Any necessary changes to the [acceptance tests](https://github.com/govuk-one-login/authentication-acceptance-tests) have been made.
+<!-- Design review
+When a new feature or front end change goes live, ensure that a design review has been performed.
+Contact a UCD colleague in your team to arrange a review.
```
Comment (mattvot) [inline .github/PULL_REQUEST_TEMPLATE.md:43]:
```suggestion
Contact a UCD colleague in the Authentication team to arrange a review.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -29,39 +27,41 @@ Include screenshots where possible, including those representing error states. H
...
 -->
+- [ ] Changes to the user interface have been demonstrated
 
-<!-- Delete this section if the PR does not change the UI. -->
-
-## Performance Analysis have been informed of the change
-
-- [ ] Performance Analysis have been informed of the change
+<!-- Performance analysis
+Make sure that a performance analyst colleague in your team has been informed of any changes to the user interfaces or user journeys.
```
Comment (gtvj) [inline .github/PULL_REQUEST_TEMPLATE.md:35]: Hi @mattvot - just wondering if we should say Authentication team here too

**Comment by gtvj** [comment]
> Hi @gtvj
> 
> > just wondering if the two UCD review items might be combined ("changes to the user interface have been demonstrated" and "a design review has been performed") as they seem broadly similar. Apologies if I'm missing some context here.
> 
> I think the intention with them is slightly different. The first is about making sure that when you are making changes that are different from the design (be it hidden by feature flagged or not), that UCD know about it and can update the design files. The seconds is more about a final check before something is seen in live in production that it passed a design review.
> 

Thanks @mattvot - it's probably just me, but I'm still a bit confused about the difference between the two and when I'd need to do either one (or both). Some changes will immediately change the journey or user interface, but others will be feature flagged. In both cases, I'd imagine the changes introduced by a PR would ideally need to be reviewed from a UCD perspective once to ensure that things appear and work as they should (and so that Figma can be updated). Is the design review maybe only for PRs that are the "final piece in the puzzle" (for example, at the point of enabling a feature flag), and the "changes to the user interface" one for one-shot changes? Once again, sorry - but I'm not sure I understand and worry a bit UCD colleagues might end up being asked to look at the same thing twice.

**Comment by mattvot** [comment]
Thanks for the feedback @gtvj, as you've seen I've brought the conversation internal while we discuss with UCD

**Pull request merged**